### ISSUE TITLE
Bad Alignment

### ISSUE THREAD

**Issue by nich3712** [keithito/tacotron#126]
After 200000 steps, my alignment is still poor and I'm not sure why. I have linked my parameters so that you can see if there are any glaring issues and I am also using the ljspeech 1.1 dataset. Does anybody know what I'm doing wrong?

import tensorflow as tf
​
​
# Default hyperparameters:
hparams = tf.contrib.training.HParams(
 # Comma-separated list of cleaners to run on text prior to training and eval. For non-English
 # text, you may want to use "basic_cleaners" or "transliteration_cleaners" See TRAINING_DATA.md.
 cleaners='english_cleaners',
​
 # Audio:
 num_mels=80,
 num_freq=1025,
 sample_rate=20000,
 frame_length_ms=50,
 frame_shift_ms=12.5,
 preemphasis=0.97,
 min_level_db=-100,
 ref_level_db=20,
​
 # Model:
 # TODO: add more configurable hparams
 outputs_per_step=5,
​
 # Training:
 batch_size=32,
 adam_beta1=0.9,
 adam_beta2=0.999,
 initial_learning_rate=0.002,
 decay_learning_rate=True,
 use_cmudict=False, # Use CMUDict during training to learn pronunciation of ARPAbet phonemes
​
 # Eval:
 max_iters=200,
 griffin_lim_iters=60,
 power=1.5,       # Power to raise magnitudes to prior to Griffin-Lim
)
​
​
def hparams_debug_string():
 values = hparams.values()
 hp = [' %s: %s' % (name, values[name]) for name in sorted(values)]
 return 'Hyperparameters:\n' + '\n'.join(hp)

**I am kind of new at this so any help or suggestion would much appreciated**

![step-208000-align](https://user-images.githubusercontent.com/15917398/37298310-4ec3af9c-25f6-11e8-9c86-9f7f1bfb0356.png)

**Comment by keithito** [comment]
@nich3712: can you share your loss curve? It's possible you ended something bad happened earlier on in training and it couldn't recover.

**Comment by abuvaneswari** [comment]
Hello @keithito ,

Great repository here! And the demo_server is an added plus. The pretrained model you shared works quite ok. But I am having problems getting my own training to learn a good alignment.

The hparams are the same as what @nich3712 shared above. I am at step 297k. 

Alignment image
![step-297000-align](https://user-images.githubusercontent.com/15253225/40247538-c2647d22-5a9a-11e8-9512-8c1fea1f9787.png)

Training log
[tacotron_Keithito.train.zip](https://github.com/keithito/tacotron/files/2017776/tacotron_Keithito.train.zip)

I understand I may have to restart the training. 

Can you please let me know at approximately which #step I should expect to see a decent alignment?

Thanks a lot!
-Buvana

**Comment by keithito** [comment]
@abuvaneswari On the LJ speech dataset, it's generally around step 20,000 when you start getting a decent alignment.
